src/snippets/class.md

```diff
@@ -0,0 +1,58 @@
+---
+description: Class / Classes Snippet
+---
+
+# Class
+
+A snippet on using Classes in AssemblyScript, and assigning instance vs. static properties.
+
+```editor
+#!optimize=size&runtime=full
```

Comment:
```suggestion
#!runtime=full
```

As in the earlier comment, this compiles a little faster due to skipping `--converge`. I guess using the full runtime here is reasonable, as we are dealing with managed objects here under the hood, even though the HTML does not receive these. Perhaps retaining/releasing a managed object, preferably a string so it doesn't become too complex, would be a good last case?

Reply:
Ah, so I'm a little lost on what you mean here regarding the "good last case in the HTML" :upside_down_face: . But, I'm down for whatever suggestion helps show off the syntax / pattern for using classes :smile: :joy: